### PR TITLE
Retry switch fallthrough fix and create service tests

### PR DESCRIPTION
**Pull request by rdelval**
* Added tests for create service
* Added test for catching an invalid executor.
* Passed response object back to user whenever possible for CreateJob, StartJobUpdate, and CreateService.

### CODE REVIEW

File: retry.go [hunk 1]
```diff
@@ -178,18 +177,21 @@ func (r *realisClient) thriftCallWithRetries(thriftCall auroraThriftCall) (*auro
...
-				// If the response code is transient, continue retrying
+			// If the response code is transient, continue retrying
 			case aurora.ResponseCode_ERROR_TRANSIENT:
 				r.logger.Println("Aurora replied with Transient error code, retrying")
 				continue
 
-				// Failure scenarios, these indicate a bad payload or a bad config. Stop retrying.
+			// Failure scenarios, these indicate a bad payload or a bad config. Stop retrying.
 			case aurora.ResponseCode_INVALID_REQUEST:
+				fallthrough
```
Comment (kkrishna) [inline retry.go]: why not use case list instead of fallthrough?

Comment (rdelval) [reply retry.go]: Can you give me an example of what this would look like?

Comment (rdelval) [reply retry.go]: Just saw what this looks like, I'll update the PR

Comment (at15) [reply retry.go]: you can use multiple condition in one case 

````go
switch nitendo {
case Mario, Zelda:
      // ....
}
````

see playground https://play.golang.org/p/XHnWbHeq814

**Pull request merged**